Braintree.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -6,7 +6,7 @@
       "location" : "https://github.com/paypal/paypalcheckout-ios/",
       "state" : {
         "revision" : "773568fbbffd54f900d6d78be7793ae1871d3b35",
```

Comment:
Based on [this previous PR](https://github.com/braintree/braintree_ios/pull/1073/files) the revision hash should be updated as well. Sometimes SPM doesn't play nice and update automagically as it's supposed to. If you open `Braintree.xcodeproj` instead of the `.xcworkspace` it should update this file for you as expected.

Reply:
I see these changes now:
![image](https://github.com/braintree/braintree_ios/assets/136759951/b482c5bf-5d17-419a-bb86-706fa2e74f38)
should I commit both files or just package.resolved?

Reply:
Only the `Package.resolved` should be needed - not sure why there is a diff with the `xcworkspacedata`